### PR TITLE
Formatting and spelling updates

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

File: v1-migration.md [hunk 1]
```diff
@@ -117,7 +115,7 @@ The eCom v2 API is available at ```ecomm/v2/payments```:
 
 ## Documentation
 
-The eCom v2 API documentation is available here: https://github.com/vippsas/vipps-ecom-api
+The eCom v2 API documentation is available here: <https://github.com/vippsas/vipps-ecom-api>
```
Comment (cloveras) [inline v1-migration.md:118]: What do the brackets do?

File: vipps-ecom-api-faq.md [hunk 2]
```diff
@@ -332,14 +335,21 @@ product or service is provided to the customer. See:
...
 >Følger ikke betalingstiden av avtalen, skal forbrukeren betale når selgeren krever det.
 >Hvis ikke noe annet er avtalt, har forbrukeren ikke plikt til å betale kjøpesummen uten at tingen blir overlevert eller stilt til hans eller hennes rådighet i samsvar med avtalen og loven.
 >Forbrukeren er ikke bundet av en forhåndsavtale om plikt til å betale på et bestemt tidspunkt uavhengig av om selgeren oppfyller til rett tid.
 
+This roughly translates to:
+
+>If the payment time does not follow the agreement, the consumer must pay when the seller demands it.
+>If nothing else has been agreed, the consumer has no obligation to pay the purchase price without the item being handed over or placed at his or her disposal in accordance with the agreement and the law.
+>The consumer is not bound by a prior agreement on the obligation to pay at a specific time regardless of whether the seller fulfils on time.
+
```
Comment (cloveras) [inline vipps-ecom-api-faq.md:348]: Nice

**Review by cloveras — APPROVED** [review]
Goodie! Se my bracket comment, otherwise: Fire at will!

File: v1-migration.md (hunk 1 above)
Comment (rebekaburnett) [reply v1-migration.md:118]: The brackets just get rid of a warning against "bare urls".

**Pull request merged**